src/config/dev.ts

```diff
@@ -0,0 +1,7 @@
+export const devConfig = {
+  disableGetIndex: false, // Make cozy-app hosted by webpack-dev-server work with HTTPServer
+  forceHideSplashScreen: false, // Hide react-native splash screen renders in development mode,
+  ignoreLogBox: false, // Hide react-native logbox renders in development mode,
+  isOffline: false, // Force offline mode in development mode,
```

Comment:
I would prefer `forceOffline` to be consistent with `forceHideSplahScreen` but also to avoid confusion in code when reading `if (isOffline)` (i.e. if false, then we are online)